{"name": "Проверка повторной регистрации пользователя, который уже зарегистрирован", "status": "passed", "description": "Создаем пользователя с данными ранее зарегистрированного пользователя,     проверяем, что код ответа: 403, и текст ответа \"User already exists\"", "start": 1722593787281, "stop": 1722593787776, "uuid": "0101767a-4ae3-4723-b05c-f26d6c080b31", "historyId": "ff2db8df07163ef3ea3a30d3800d0648", "testCaseId": "ff2db8df07163ef3ea3a30d3800d0648", "fullName": "tests.test_create_user.TestUser#test_create_the_same_user", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_user"}, {"name": "subSuite", "value": "TestUser"}, {"name": "host", "value": "InnaNB"}, {"name": "thread", "value": "27624-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_user"}]}